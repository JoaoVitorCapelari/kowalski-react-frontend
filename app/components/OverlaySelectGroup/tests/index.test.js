// import React from 'react';
// import { shallow } from 'enzyme';

// import OverlaySelectGroup from '../index';

describe('<OverlaySelectGroup />', () => {
  it('Expect to have unit tests specified', () => {
    expect(true).toEqual(false);
  });
});
