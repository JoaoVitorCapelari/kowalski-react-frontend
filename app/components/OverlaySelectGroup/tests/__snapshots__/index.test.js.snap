// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<OverlaySelectGroup /> renders 1`] = `"<div class=\\"OverlaySelectGroup__OverlaySelectGroupContainer-iMhecZ iNNeSU\\"></div>"`;
